models/deepseek/v4/prefill_layer.py

```
@@ -306,33 +305,28 @@ def prefill_layer_core(
         # across the request's tiles via in-place mutation by the children).
         kv_cache_req = pl.slice(kv_cache, [ORI_CACHE_BLOCKS, BLOCK_SIZE, 1, HEAD_DIM],
                                 [ridx * ORI_CACHE_BLOCKS, 0, 0, 0])
-        ori_block_table_req = pl.slice(ori_block_table, [ORI_TABLE_BLOCKS], [ridx * ORI_TABLE_BLOCKS])
+        ori_block_table_req = pl.slice(ori_block_table, [B, ORI_TABLE_BLOCKS], [ridx * B, 0])
```

Ruff (0.15.20)

[error] Undefined name `B`

(F821)